lib/beaker-hostgenerator/cli.rb

```diff
@@ -66,6 +68,14 @@ def initialize(argv = ARGV.dup)
           @options[:list_supported_values] = true
         end
 
+        opts.on('--extract-templates-from-file HOSTS_FILE') do |f|
+          @options[:extract_templates] = f
+        end
+
+        opts.on('--templates-only') do
+          @options[:templates_only] = true
+        end
```

Comment:
I'd imagine so, yeah.  For now, I was just copying exactly what @joshcooper had.